apps/src/templates/studioHomepages/teacherHomepageV2/CourseContentDropdown.tsx

```diff
@@ -1,24 +1,104 @@
+import {SimpleDropdown} from '@code-dot-org/component-library/dropdown';
 import {BodyThreeText} from '@code-dot-org/component-library/typography';
-import React from 'react';
+import React, {useEffect, useState, useMemo} from 'react';
+import {useNavigate} from 'react-router-dom';
 
 import {Section} from '@cdo/apps/templates/teacherDashboard/types/teacherSectionTypes';
+import HttpClient from '@cdo/apps/util/HttpClient';
 import i18n from '@cdo/locale';
 
+import {TEACHER_NAVIGATION_SECTIONS_URL} from '../../teacherNavigation/TeacherNavigationPaths';
+
+import {TaskButton} from './TaskButton';
+
 import styles from './teacherHomepage.module.scss';
 
 interface CourseContentDropdownProps {
   section: Section;
 }
 
+interface UnitLessons {
```

Comment:
how about calling them UnitLessonOptions, or adding a comment that explicitly says these will be values in a dropdown?